src/Components/Patient/ManagePatients.tsx

```diff
@@ -902,7 +902,7 @@ export const PatientManager = () => {
             <CountBlock
               text="Total Patients"
               count={data?.count || 0}
-              loading={isLoading}
+              loading={(!data?.count && data?.count !== 0) || isLoading}
```

Comment:
@FrankenDeba make this change at CountBlock so that it is applied throughout the application